src/record_syscall.cc

```diff
@@ -2776,8 +2852,9 @@ static Switchable rec_prepare_syscall_arch(RecordTask* t,
           1, ParamSize::from_syscall_result<int>((size_t)t->regs().arg2()));
       return (GRND_NONBLOCK & t->regs().arg3()) ? PREVENT_SWITCH : ALLOW_SWITCH;
 
+    case Arch::get_thread_area:
     case Arch::set_thread_area:
-      syscall_state.reg_parameter<typename Arch::user_desc>(1, IN_OUT);
```

Comment:
`::user_desc` is used in most other places (such that the same method can be used for both archs), so it just seemed more uniform. Happy to change it back though.